File: 프로그래머스/unrated/181865. 간단한 식 계산하기/간단한 식 계산하기.java
class Solution {
    public int solution(String binomial) {
        int answer = 0;
        String[] str = binomial.split(" ");
        
        if (str[1].equals("+")) {
            answer = Integer.valueOf(str[0]) + Integer.valueOf(str[2]);
        }
        else if (str[1].equals("-")) {
            answer = Integer.valueOf(str[0]) - Integer.valueOf(str[2]);
        }
        else if (str[1].equals("*")) {
            answer = Integer.valueOf(str[0]) * Integer.valueOf(str[2]);
        }
        return answer;
    }
}

File: 프로그래머스/unrated/181865. 간단한 식 계산하기/간단한 식 계산하기.py
def solution(binomial):
    array = binomial.split()
    
    a = int(array[0])
    op = array[1]
    b = int(array[2])
    
    if op=="+":
        return a+b
    elif op=="-":
        return a-b
    else:
        return a*b
    